brave/brave-core
# Add Rewards page promo card for Brave Creators

Resolves https://github.com/brave/brave-browser/issues/25503

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on
  
## UI:

![Screenshot 2022-10-25 112831](https://user-images.githubusercontent.com/5033691/197849258-69b26368-c1ce-4ea4-b42f-e231df6a79bb.png)

## Test Plan:

- Clean profile
- Enable Rewards
- Verify that Brave Creators promo card appears in first promo position
- Verify that clicking Brave Creators promo card opens up a tab with https://creators.brave.com
- Verify that closing the promo card deletes it and persists deletion across browser restart
